### PR TITLE
Fragments

### LINKED ISSUE #11: Implemented Fragment task (team bootcamp)

### PR DESCRIPTION
**Pull request by damiengenoud**
Implements an Activity with a Drawer Menu that enables switching between different Fragments.
closes #11

### CODE REVIEW

File: app/src/androidTest/java/com/github/drawyourpath/bootcamp/FragmentActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        onView(withId(R.id.mainTextView)).check(matches(withText("Main Fragment")))
+
+        //Profile fragment
+        launchFragmentInContainer<ProfileFragment>()
+        onView(withId(R.id.profileTextView)).check(matches(withText("Profile Fragment")))
+
+        //Settings fragment
+        launchFragmentInContainer<SettingsFragment>()
+        onView(withId(R.id.settingsTextView)).check(matches(withText("Settings Fragment")))
+    }
```
Comment (HugoHof) [inline app/src/androidTest/java/com/github/drawyourpath/bootcamp/FragmentActivityTest.kt:59]: The code is clean and the comments are very usefull to understand the code.

File: app/src/main/res/layout/activity_main_drawer_header.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        android:text="Username"
+        android:textSize="34sp" />
+
+    <TextView
+        android:id="@+id/header_email"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
+        android:gravity="center"
+        android:text="example@smth.com"
```
Comment (HugoHof) [inline app/src/main/res/layout/activity_main_drawer_header.xml:24]: You can extract the text to use it as a resource.(@string/name_of_the_string).

File: app/src/main/res/layout/fragment_main.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/mainTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Main Fragment"
```
Comment (HugoHof) [inline app/src/main/res/layout/fragment_main.xml:11]: You can extract the text to use it as a resource.(@string/name_of_the_string).

File: app/src/main/res/layout/fragment_options.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/settingsTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Settings Fragment"
```
Comment (HugoHof) [inline app/src/main/res/layout/fragment_options.xml:11]: You can extract the text to use it as a resource.(@string/name_of_the_string).

File: app/src/main/res/layout/fragment_profile.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/profileTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Profile Fragment"
```
Comment (HugoHof) [inline app/src/main/res/layout/fragment_profile.xml:11]: You can extract the text to use it as a resource.(@string/name_of_the_string).

**Review by HugoHof — APPROVED** [review]

**Comment by damiengenoud** [comment]
Not needed

**Pull request closed without merging**